### DISCUSSION TITLE
Is ESS Assistant required?

### DISCUSSION THREAD

**Discussion by mlauss2** [Louisvdw/dbus-serialbattery#989] [General]
With dbus-serialbattery active, is the Victron ESS Assistant even required? (A problem I have with the ESS Assistant is that it does not accomodate my 12S Li-Ion batteries due to different working voltage ranges).

Thanks!
     Manuel

**Comment by mr-manuel** [comment]
I'm not sure, that you fully understood what the ESS assistant does. Did you read https://www.victronenergy.com/live/ess:start?

But to answer basically your question, no the ESS assistant is not needed to run dbus-serialbattery.

**Comment by mlauss2** [comment]
Yeah, I was not thinking of the whole picture.  Apologies.
